src/modules/esl-base-popup/core/esl-base-popup.ts

```diff
@@ -43,38 +40,36 @@ export class ESLBasePopup extends ESLBaseElement {
   @boolAttr() public closeOnEsc: boolean;
   @boolAttr() public closeOnOutsideAction: boolean;
 
-  @jsonAttr<PopupActionParams>({defaultValue: {silent: true, force: true, initiator: 'init'}})
+  @jsonAttr<PopupActionParams>({defaultValue: {silent: false, force: true, initiator: 'init'}})
   public initialParams: PopupActionParams;
   @jsonAttr<PopupActionParams>({defaultValue: {}})
   public defaultParams: PopupActionParams;
 
-  public get group() {
-    return ESLBasePopupGroup.find(this.groupName);
-  }
-
   protected attributeChangedCallback(attrName: string, oldVal: string, newVal: string) {
     if (!this.connected || newVal === oldVal) return;
     switch (attrName) {
       case 'open':
         this.toggle(this.open, Object.assign({initiator: 'attribute'}, this.defaultParams));
         break;
       case 'group':
-        oldVal && ESLBasePopupGroup.unregister(this, oldVal);
-        newVal && ESLBasePopupGroup.register(this, newVal);
+        this.$$fire('change:group',  {
```

Comment:
Event is 
```
 bubbles: true,
          composed: true,
          cancelable: true,
```
by default, and it merges all settings

Reply:
Fixed. Thanks!